pkg/porter/build.go

```diff
@@ -123,6 +125,9 @@ func (p *Porter) getUsedMixins() ([]mixin.Metadata, error) {
 
 	var usedMixins []mixin.Metadata
 	for _, installedMixin := range installedMixins {
+		if p.Manifest == nil {
```

Comment:
Did you notice which code path resulted in us trying to do a build when the manifest is nil? Seems like that shouldn't ever happen?

Reply:
Yes, I believe this was was hit in lint-related test(s) in build_test; updated the tests and removed this check.